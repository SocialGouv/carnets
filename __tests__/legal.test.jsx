import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import Page from "../src/app/legal/page";

test("Page", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
