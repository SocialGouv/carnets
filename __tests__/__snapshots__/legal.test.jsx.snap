// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Page 1`] = `
<div>
  <main
    class="legal"
  >
    <h2
      class="fr-h2"
    >
      Mentions légales
    </h2>
    <h3
      class="fr-h3"
    >
      Editeur de la plateforme
    </h3>
    <p
      class="fr-text--md"
    >
      La Plateforme est éditée par la 
      <a
        href=""
      >
        Fabrique des Ministères
      </a>
       
      sociaux située:
    </p>
    <address
      class="my-6"
    >
      Tour Mirabeau
      <br />
      39-43 Quai André Citroën
      <br />
      75015 PARIS
      <br />
      Tél : 01 40 56 60 00
    </address>
    <h3
      class="fr-h3"
    >
      Directeur de la publication
    </h3>
    <p
      class="fr-text--md"
    >
      Anne Jeanjean - Directrice du Numérique
    </p>
    <h3
      class="fr-h3"
    >
      Hébergement de la plateforme
    </h3>
    <p
      class="fr-text--md"
    >
      Ce site est hébergé par 
      <strong>
        OVH
      </strong>
       (région France centre) :
    </p>
    <p
      class="fr-text--md"
    >
      <i>
        OVH SAS Siège social : 2 rue Kellermann - 59100 Roubaix - France.
      </i>
    </p>
    <h3
      class="fr-h3"
    >
      À savoir
    </h3>
    <p
      class="fr-text text-justify"
    >
      Le site ne récolte ni ne traite aucune donnée à caractère personnel. La mesure d'audience (nombre de visites, pages consultées) est réalisée par un outil libre intitulé
       
      <a
        href="https://fr.matomo.org/"
        target="_blank"
      >
        Matomo
      </a>
       
      spécifiquement paramétré, respectant les conditions d'exemption du consentement de l'internaute définies par la
       
      <a
        href="https://www.cnil.fr/sites/cnil/files/atoms/files/recommandation-cookies-et-autres-traceurs.pdf"
        target="_blank"
      >
        recommandation « Cookies »
      </a>
       
      de la Commission nationale informatique et libertés (CNIL). Cela signifie, notamment, que ces cookies ne servent qu'à la production de statistiques anonymes et ne permettent pas de suivre la navigation de l'internaute sur d'autres sites.
    </p>
  </main>
</div>
`;
